File: Loadshop.DomainServices/Loadshop/Services/Data/ShippingLoadCarrierGroupData.cs
using System.Collections.Generic;

namespace Loadshop.DomainServices.Loadshop.Services.Data
{
    public class ShippingLoadCarrierGroupData : LoadCarrierGroupData
    {
        public string ShippingLoadCarrierGroupDisplay { get; set; }

        public List<LoadCarrierGroupEquipmentData> LoadCarrierGroupEquipment { get; set; } = new List<LoadCarrierGroupEquipmentData>();
        public List<CarrierData> Carriers { get; set; }
    }
}

File: Loadshop.Web.UI/src/app/shared/models/shipping-load-carrier-group-data.ts
import { Carrier } from '.';
import { defaultLoadCarrierGroup, LoadCarrierGroupData } from './load-carrier-group-data';

export interface ShippingLoadCarrierGroupData extends LoadCarrierGroupData {
  shippingLoadCarrierGroupDisplay: string;
  carriers: Carrier[];
}

export const defaultShippingLoadCarrierGroup: ShippingLoadCarrierGroupData = {
  ...defaultLoadCarrierGroup,
  shippingLoadCarrierGroupDisplay: null,
  carriers: [],
};
